app/services/concerns/xccdf/profile_rules.rb

```diff
@@ -24,20 +24,20 @@ def save_profile_rules
       def profile_rules
         @profile_rules ||= @op_profiles.flat_map do |op_profile|
           profile_id = profile_id_for(ref_id: op_profile.id)
-          rule_ids_for(ref_ids: op_profile.selected_rule_ids).map do |rule_id|
+          rule_ids_for(ref_ids: op_profile.selected_rule_ids).map do |entity_id|
```

Comment:
I guess this is what you'd need to change from `selected_rule_ids`, but only if the parser supports. Also you should verify that the method you chose doesn't return anything else than rules and groups.

Reply:
I guess eventually it'll look something like:
```suggestion
          link_ids_for(ref_ids: op_profile.selected_ids).map do |entity_id|
```
Maybe you can even reuse the cached stuff somehow to make it faster.

Reply:
Yeah, after looking at the parser, it looks like `selected_rule_ids` will be all selected ids, not only the rule ids so this should contain the rule group ids already, which is good. 

Also, based on this, it looks like the selector will be a Rule, Group or a cluster of Rules and Groups.
<img width="607" alt="Screen Shot 2022-03-17 at 11 39 43 AM" src="https://user-images.githubusercontent.com/87209745/158837008-5419c136-0f23-4056-ae0c-2496c37111b8.png">
parently